jekyll/_sass/_subnav.scss

```diff
@@ -46,3 +46,29 @@ $nav-offset-subnav: $nav-offset + $subnav-offset;
     top: -$nav-offset-subnav;
   }
 }
+
+// Another set of styles specific to this project:
+
+// 1. Override hiding of subnav on mobile
+.component.subnav {
+  @media screen and (max-width: $screen-xs-max) {
+    display: block;
+  }
+}
+#subnav-placeholder {
+  @media screen and (max-width: $screen-xs-max) {
+    display: block;
+  }
+}
+
+// 2. Add style for mobile
+.component.subnav {
+  @media screen and (max-width: $screen-xs-max) {
+    font-size: $font-size-small;
+
+    li {
```

Comment:
On the smallest screens (under 370px wide) the "Images" tab wraps, reducing the margin from 24px would help
![Screen Shot 2021-03-02 at 4 26 48 PM](https://user-images.githubusercontent.com/5113432/109721076-e26cb880-7b78-11eb-83f2-3db497b98aad.png)
.